ebean-test/src/test/resources/migrationtest/dbmigration/mariadb-noprocs/1.1.sql

```diff
@@ -36,9 +36,7 @@ alter table migtest_fk_set_null drop foreign key fk_migtest_fk_set_null_one_id;
 alter table migtest_fk_set_null add constraint fk_migtest_fk_set_null_one_id foreign key (one_id) references migtest_fk_one (id) on delete restrict on update restrict;
 
 update migtest_e_basic set status = 'A' where status is null;
-alter table migtest_e_basic alter status set default 'A';
-alter table migtest_e_basic modify status varchar(1) not null;
```

Comment:
modify will erase previously set default value, so we do it in one statement now